elixir/apps/domain/lib/domain/network.ex

```diff
@@ -4,7 +4,7 @@ defmodule Domain.Network do
 
   @cidrs %{
     # Notice: those are also part of "resources_account_id_cidr_address_index" DB constraint
-    ipv4: %Postgrex.INET{address: {100, 64, 0, 0}, netmask: 10},
+    ipv4: %Postgrex.INET{address: {100, 64, 0, 0}, netmask: 11},
     ipv6: %Postgrex.INET{address: {64_768, 8_225, 4_369, 0, 0, 0, 0, 0}, netmask: 106}
```

Comment:
Maybe a good idea to update the IPv6 range too to match?
```suggestion
    ipv6: %Postgrex.INET{address: {64_768, 8_225, 4_369, 0, 0, 0, 0, 0}, netmask: 107}
```

Reply:
Might be more consistent in docs to say "we reserve 2 million addresses for internal use in Firezone. `IPv4: x.x.x.x/11, IPv6: x:x:x:x/107`"